### ISSUE TITLE
Fail to add nwchem app kernel due to that mod_appkernel_app_kernel_def is NOT iterable

### ISSUE THREAD

**Issue by weijianwen** [ubccr/akrr#9]
Hi,

I built akrr 2.0.0-1 RPM from the latest commit on github. `akrr setup`, `akrr resource add` and `akrr deploy` work well. But issues arise when running `akrr app add -a nwchem` to add NWChem app kernel which said TypeError: 'int' object is not iterable. I add two lines into `generate_tables.py` displaying content and type of `mod_appkernel_app_kernel_def`. The step to reproduce this issue are as follows. I appreciate if any suggestion is given.

```
[root@jobs sjtupi]# akrr app add -a nwchem -r sjtupi --dry-run
[INFO] Generating application kernel configuration for nwchem on sjtupi
(24, 'NWChem', 'nwchem', 'node', 0, '<a href="http://www.nwchem-sw.org" target="_blank" alt="nwchem">NWChem</a> is an ab initio computational chemistry software package developed by Pacific Northwest National Laboratory.\r\n<p>\r\nThe input to the benchmark runs is the Hartree-Fock energy calculation of Au+ with MP2 and Coupled Cluster corrections.\r\n<p>\r\nThe version of NWChem being benchmarked is 5.1.1.\r\n<p>\r\nThe metrics we show here contain NWChem\'\'s self-collected Global Arrays statistics. The Global Arrays toolkit is the communication library used by NWChem to manipulate large akrrays distributed across compute nodes. The Global Arrays toolkit has three basic operations: Get (fetch values from remote memory), Put (store values to remote memory), and Accumulate (update values in remote memory). NWChem measures the numbers of these operations and the amount of data affected by them.', 0, None)
<class 'tuple'>
Traceback (most recent call last):
  File "/usr/bin/akrr", line 65, in <module>
    akrr.cli.CLI().run()
  File "/usr/lib/python3.6/site-packages/akrr/cli/__init__.py", line 157, in run
    return cli_args.func(cli_args)
  File "/usr/lib/python3.6/site-packages/akrr/cli/commands.py", line 414, in handler
    app_add(args.resource, args.appkernel, args.execution_method, args.dry_run)
  File "/usr/lib/python3.6/site-packages/akrr/app.py", line 64, in app_add
    mod_appkernel_app_kernel_def=appcfg['db_setup']["mod_appkernel_app_kernel_def"])
  File "/usr/lib/python3.6/site-packages/akrr/cli/generate_tables.py", line 492, in populate_mod_appkernel_app_kernel_def
    mod_appkernel_app_kernel_def:
TypeError: 'int' object is not iterable
```

**Comment by nsimakov** [comment]
Hello,
Thanks for the report, I will check it. On a side note AKRR should be run as regular user.
Regards,
Nikolay

**Comment by nsimakov** [comment]
Yep, I can reproduce it, working on fix.

**Comment by nsimakov** [comment]
Ok, it is fixed in 52926b66f7142114ddbc613b211bb36256cc5b8b. You need to update source code (uninstall old rpm, make and install new one) and continue from where you stoped.

**Comment by weijianwen** [comment]
Thank you. I'll rebuid the RPM to confirm if this issue persists.
